Model/Button/GetCartButtonConfig.php

```diff
@@ -274,9 +274,21 @@ private function getOrderContent($quote): array
                 'quantity' => $item->getQty(),
                 'total' => (float)$item->getRowTotalInclTax(),
                 'url' => $item->getProduct()->getProductUrl(),
-                'image' => $this->storeManager->getStore()->getBaseUrl(UrlInterface::URL_TYPE_MEDIA) . 'catalog/product' . $item->getProduct()->getImage(),
+                'image' => $this->getProductImageUrl($item->getProduct())
             ];
         }
         return $orderContent;
     }
+
+    /**
+     * @param $product
+     * @return string
+     */
+    private function getProductImageUrl($product)
```

Comment:
I think we can add type hints here (and in the docblock)
```suggestion
    private function getProductImageUrl(ProductInterface $product): string
```